.github/scripts/solvers.sh

```diff
@@ -3,6 +3,7 @@
 . .github/scripts/preamble.sh
 
 export OPAMYES=1
```

Comment:
A very small nit, but as we set `OPAMCONFIRMLEVEL`, shall we stop setting `OPAMYES`?

Reply:
Added to the dose-7 PR: https://github.com/ocaml/opam/pull/4760/commits/05533008c535999fdb479cb05319f3f98fc8d86e